File: API/Entity/TurnarroundVehicleTimeOffset.cs
using System.ComponentModel.DataAnnotations;

namespace API.Entity
{
    public class TurnarroundVehicleTimeOffset
    {
        [Key]
        public int TvtoId { get; set; }
        public int TimeOffsetStart { get; set; }
        public int TimeOffsetEnd { get; set; }
        public int AircraftTurnarroundTemplateId { get; set; }
        public AircraftTurnarroundPreset? AircraftTurnarroundTemplate { get; set; }
        public int VehicleTypeId { get; set; }
        public VehicleType? VehicleType { get; set; }
    }
}

File: client/src/app/_models/turnarroundVehicleTimeOffset.ts
import { aircraftTurnarroundPreset } from "./aircraftTurnarroundPreset";
import { vehicleType } from "./vehicleType";

export class turnarroundVehicleTimeOffset
{
    tvtoId: number;
    timeOffsetStart: number;
    timeOffsetEnd: number;
    aircraftTurnarroundPreset: aircraftTurnarroundPreset
    vehicleTypeId: number;
    vehicleType: vehicleType;
}